src/Config.php

```diff
@@ -26,10 +22,12 @@
  *
  * These features are advanced, and not typically used. If you wish to use them,
  * you will need to require the laminas-stdlib package in your application.
+ *
+ * @psalm-import-type ServiceManagerConfigurationType from ConfigInterface
  */
 class Config implements ConfigInterface
 {
-    /** @var array */
+    /** @var array<string,bool> */
```

Comment:
Not important, but I noticed that your types have no spacing in it 😁

Reply:
why would they need spacings? is that another coding style thing? 
The more spaces, the merrier line length issues on complex types 😩

Reply:
Types are not just an afterthought - should be as readable as other stuff 😛